src/core/execute.h

```diff
@@ -145,6 +145,11 @@ typedef enum ExecCleanMask {
         _EXEC_CLEAN_MASK_INVALID = -1,
 } ExecCleanMask;
 
+typedef enum ExecSyscallErrorAction {
+        EXEC_SYSCALL_ERROR_ACTION_KILL = 0,
+        EXEC_SYSCALL_ERROR_ACTION_LOG = -1,
```

Comment:
hmm, I'd prefer not to use negative values here. Maybe use really large ones, i.e. INT32_MAX-1 or so? errnos are guaranteed on linux to be below 4K, so this should be a safe choice

Reply:
ok